login/service_whitebox_test.go

```diff
@@ -0,0 +1,151 @@
+package login
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/almighty/almighty-core/account"
+	"github.com/almighty/almighty-core/resource"
+	"github.com/almighty/almighty-core/token"
+	"github.com/jinzhu/gorm"
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/github"
+)
+
+var db *gorm.DB
+var loginService *gitHubOAuth
+
+// Github doesn't allow commiting actual tokens no matter how
+// less privleges the token has.
```

Comment:
interesting.
Does this token have any kind of validity ? e.g> If the owner of this token resets his github password, then is the token refreshed ?
Sorry but, I am just trying to make sure that test wont fail because of such reason.